share/DK.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<holidays xmlns="https://heigl.org/holidays"
+          xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+          xsi:schemaLocation="https://heigl.org/xml/xsd/holidays.xsd"
+>
+    <date day="1" month="1" free="true">Nytårsdag</date>
+    <easter offset="-7" free="true">Palmesøndag</easter>
+    <easter offset="-3" free="true">Skærtorsdag</easter>
+    <easter offset="-2" free="true">Langfredag</easter>
+    <easter offset="0" free="true">Påskedag</easter>
+    <easter offset="1" free="true">Anden påskedag</easter>
+    <easter offset="25" free="true">Store bededag</easter>
+    <easter offset="39" free="true">Kristi Himmelfartsdag</easter>
+    <easter offset="49" free="true">Pinsedag</easter>
+    <easter offset="50" free="true">Anden Pinsedag</easter>
+    <date day="5" month="6" free="true">Grundlovsdag</date>
+    <date day="24" month="12" free="true" comment="shops are closed">Juleaften</date>
+    <date day="25" month="12" free="true">Juledag</date>
+    <date day="26" month="12" free="true">Anden juledag</date>
+    <date day="31" month="12" free="true" comment="Shops close at 3PM">Nytårsaften</date>
```

Comment:
I don't know if this is just a pointer, but there are no public rules to my knowledge when the shop closes. 
Sorry for jumping in. Just saw Scandinavia was added, and as a Dane I was curious.

Reply:
Thanks for taking that up. I took the info from https://en.wikipedia.org/wiki/Public_holidays_in_Denmark (which cites https://www.retsinformation.dk/eli/lta/2019/515) which states that most shops close by law from 3pm. 

My Danish is – euphemistically put – rather rusty, so I can't really say whether that's correct. Should I rather remove that comment then?

Reply:
Google Translate though actually translates §2 klike this:

> § 2. No retail sales may take place from the stores mentioned in § 1, para. 1, on public holidays, Constitution Day and Christmas Eve and after kl. 15 on New Year's Eve.

I interpret this kl. 15 as 3PM....

Reply:
That's also a correct translation then. Was just unknown to me. 
And retsinformation.dk is a valid source :)

Reply:
I would leave the comment as it's correct according to retsinformation.dk